modules/ROOT/partials/configuration/tinycomments_can_delete.adoc

```diff
@@ -8,9 +8,9 @@ Type : `+Function+`
 Default Function :
 [source,js]
 ----
-function (req, done, fail) {
-  var allowed = req.comments.length > 0 &&
-                req.comments[0].author === <Current_tinycomments_author>;
+(req, done, fail) => {
+  const allowed = req.comments.length > 0 &&
+                req.comments[0].author === '<Current_tinycomments_author>';
```

Comment:
```suggestion
                req.comments[0].author === <Current_tinycomments_author>;
```
This shouldn't have quotes as it's meant to be a variable replacement, so adding quotes makes it appear to be an actual string that maybe shouldn't be replaced.